examples/lerax-square.cpp

```diff
@@ -1,16 +1,30 @@
 #include <iostream>
 #include <cmath>
 #include <string>
+#include <map>
 
 #include <SDL2/SDL.h>
 
 #include "math/Vector2.hpp"
+#include "GameLoop.hpp"
+#include "InputHandler.hpp"
 
 #define SCREEN_WIDTH 800
 #define SCREEN_HEIGHT 600
 
 using namespace std;
 
+// keys to InputHandler observe
+std::map<const char*, SDL_Keycode> input_config = {
+    {"quit", SDLK_ESCAPE},
+    {"up", SDLK_UP},
+    {"left", SDLK_LEFT},
+    {"right", SDLK_RIGHT},
+    {"down", SDLK_DOWN},
+    {"velocity_up", SDLK_w},
+    {"velocity_down", SDLK_s},
+};
+
 struct KeyboardState {
```

Comment:
Why is this here still?

Reply:
This is just a collection of state that InputHandler will save the result on it. I'm not providing Player direct access to InputHandler, rather that I'm processing the events on `Game::event` and saving on KeyboardState.

How do you think I can improve it?

Reply:
Let's discuss everything which is about hazard-memory lib in the issue space.